src/Workspaces/CSharp/Portable/Formatting/CSharpFormattingOptions.cs

```diff
@@ -4,145 +4,237 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Formatting
 {
-    public static class CSharpFormattingOptions
+    public static partial class CSharpFormattingOptions
     {
         public static Option<bool> SpacingAfterMethodDeclarationName { get; } = new Option<bool>(nameof(CSharpFormattingOptions), nameof(SpacingAfterMethodDeclarationName), defaultValue: false,
-            storageLocations: new RoamingProfileStorageLocation("TextEditor.CSharp.Specific.SpacingAfterMethodDeclarationName"));
+            storageLocations: new OptionStorageLocation[] {
```

Comment:
`EditorConfigStorageLocation`  and `RoamingProfileStorageLocation` are both derived from `OptionStorageLocation` but implicit array creation cannot infer `OptionStorageLocation`